pkg/cloudprovider/providers/gce/gce_networkendpointgroup.go

```diff
@@ -0,0 +1,151 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package gce
+
+import (
+	"strings"
+	"time"
+
+	computealpha "google.golang.org/api/compute/v0.alpha"
+)
+
+const (
+	ExperimentalKey              = "X-Goog-Experiments"
```

Comment:
Are we supposed to leak this to OSS? It may be useful to keep that as an option to gce.conf